boxes/static/javascripts/box_list.js

```diff
@@ -2,24 +2,27 @@ $(document).ready(function() {
   /*
     On click copy the box link to the clipboard
   */
+  $(".box-options").click(function(){
+    $(this).submit();
+  });
   $(".copy-to-clipboard-js").on("click", function(){
     $this = $(this);
-    var previousText = $this.text();
+    var previousText = $this.html();
```

Comment:
The text() doesn't work because i want the icon to come back after clicking the button to copy the url. And before it was just a text saying "copy link" and it worked, but now it's an image so I just used the html() to restore the original html doc and that makes the icon reappears.